### ISSUE TITLE
Data Augmentation

### ISSUE THREAD

**Issue by carlosd-29** [carlosd-29/AudioClassification#8]
Perform some time-frequency augmentation to prevent overfitting

- [ ] Frequency mask
- [ ] Time mask
- [x] Time shift
- [x] Pitch shift
- [ ] Time Stretch
- [x] Add noise
- [ ] Re-train

**Comment by carlosd-29** [comment]
Augmented data by adding nosy, performing a time shift and pitch shift. This brought training data up x4. It's in the process of retraining

**Comment by carlosd-29** [comment]
Retraining: can close and move on to #6
